codebuild_specs/run_canary_e2e_tests.yml

```diff
@@ -9,7 +9,17 @@ env:
     NODE_OPTIONS: --max-old-space-size=8096
 phases:
   build:
+    commands:
+      - codebuild-breakpoint
+      - source ./shared-scripts.sh && _setupE2ETestsLinux
+      - source ./shared-scripts.sh && _runE2ETestsLinux
+  post_build:
     commands:
       - source ./shared-scripts.sh && _unassumeTestAccountCredentials
       - aws sts get-caller-identity
-      - source ./shared-scripts.sh && _emitCreateApiCanaryMetric
+      - source ./shared-scripts.sh && _scanArtifacts && _emitCreateApiCanaryMetric
```

Comment:
Can you confirm that `_emitCreateApiCanaryMetric` includes the region as a metric dimension?

Reply:
I've added region to metric dimension.